codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/FullyQualifiedNameExtractorSuite.scala

```diff
@@ -61,6 +61,9 @@ class FullyQualifiedNameExtractorSuite extends munit.FunSuite {
     val fqn = fqnExtractor(stateCart)
     assertNoDiff(fqn.name, "Cart")
     assertNoDiff(fqn.parent.scalaPackage, "com.example.shoppingcart.domain.shoppingcart_domain")
+    assertNoDiff(
```

Comment:
I'm not sure what the difference of `assertNoDiff` and `assertEquals` is, but this looks like it should be an equals?

Reply:
https://scalameta.org/munit/docs/assertions.html#assertnodiff

> The difference between assertNoDiff() and assertEquals() is that assertEquals() fails according to the == method while assertNoDiff() ignores non-visible differences such as trailing/leading whitespace, Windows/Unix newlines and ANSI color codes

Hmm, maybe we want `assertEquals` always after all?